sdk/storage/storage-file-datalake/samples-dev/dataLakeServiceClient.ts

```diff
@@ -44,49 +47,51 @@ export async function main() {
     sharedKeyCredential
   );
 
-  let i = 1;
+  console.log("Filesystems:");
   for await (const filesystem of serviceClient.listFileSystems()) {
-    console.log(`FileSystem ${i++}: ${filesystem.name}`);
+    console.log(`- ${filesystem.name}`);
   }
 
   // Create a filesystem
-  const fileSystemName = `newfilesystem${new Date().getTime()}`;
-  const fileSystemClient = serviceClient.getFileSystemClient(fileSystemName);
+  const filesystemName = `newfilesystem${new Date().getTime()}`;
```

Comment:
What naming convention we should follow here?  
Usually, file and system should be two separated words, so system should be capitalized.

Reply:
Other azure storage documentation uses "filesystem" as a single word, and this is how I'm used to seeing it written in linux manual pages, etc.

As far as I understand, it may be written correctly either way.

Reply:
Example: https://docs.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/filesystem

Reply:
Though I suppose because the client itself separates them (`getFileSystemClient`) we should at least be internally consistent within the JS samples.

Reply:
However, a lot of the implementation also uses `filesystem` instead of `fileSystem` in property names, the README, etc.

This should be your call @EmmaZhu : whichever you think is right for the samples I'll change it to that before merging.

Reply:
It may be better to first keep consistent on client, so I would vote for `fileSystem`.